src/main/java/ita/softserve/course_evaluation/exception/handler/GlobalControllerExceptionHandler.java

```diff
@@ -157,17 +157,8 @@
 
     @Override
     protected ResponseEntity<Object> handleMethodArgumentNotValid(MethodArgumentNotValidException exception, HttpHeaders headers, HttpStatus status, WebRequest request) {
-
```

Comment:
Please don't remove it!

Reply:
Done. Return ValidationExceptionResponse